src/main/java/de/cronn/validation_files_diff/impl/MyRefactoringElementListenerProviderImpl.java

```diff
@@ -0,0 +1,89 @@
+package de.cronn.validation_files_diff.impl;
+
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.module.ModuleManager;
+import com.intellij.openapi.module.ModuleUtil;
+import com.intellij.openapi.vfs.LocalFileSystem;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.openapi.vfs.VirtualFileManager;
+import com.intellij.psi.*;
+import com.intellij.refactoring.listeners.RefactoringElementAdapter;
+import com.intellij.refactoring.listeners.RefactoringElementListener;
+import com.intellij.refactoring.listeners.RefactoringElementListenerProvider;
+import com.intellij.refactoring.rename.RenameProcessor;
+import com.intellij.refactoring.rename.RenamePsiElementProcessor;
+import com.intellij.refactoring.rename.RenamePsiFileProcessor;
+import com.intellij.usageView.UsageInfo;
+import de.cronn.validation_files_diff.ValidationDiffProjectOptionsProvider;
+import de.cronn.validation_files_diff.helper.ModuleAnalyser;
+import de.cronn.validation_files_diff.helper.PsiTestNameUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class MyRefactoringElementListenerProviderImpl implements RefactoringElementListenerProvider {
+	@Override
+	public @Nullable RefactoringElementListener getListener(PsiElement element) {
+		if (!(element instanceof PsiMethod)) {
```

Comment:
Sometime we might want to react as well on renamings of the tests classes.

Reply:
Yes, true done 7bb5a0e12dbf2bb75ea1f3bd5907ed44e0afc827

Reply:
yes, true. done 7bb5a0e12dbf2bb75ea1f3bd5907ed44e0afc827